### ISSUE TITLE
Fix color contrast on “Following” button in /onboarding flow

### ISSUE THREAD

**Issue by pkfrank** [forem/forem#19358]
## Describe the bug

The “Suggested people to follow” slide in the `/onboarding` flow has some issues with color contrast.  When the user is selected to be followed, the “Following” text does not have sufficient contrast.

This is both visually unappealing and an a11y issue.

## To Reproduce

1. Go to `/onboarding`
2. Proceed to “Suggested people to follow” slide
3. Select all users
4. Observe that the “Following” button doesn’t have enough contrast

## Expected behavior

We should use the appropriate color with enough contrast.

## Screenshots

![example](https://p-DgFRWr8y.t2.n0.cdn.getcloudapp.com/items/p9ueY2ol/93a29bd7-c66a-4c4f-aecb-199887580d2b.jpg?v=467821865edaa0c9f2a54668b69d411d)

## Additional context

The button color here is taken from Admin > Config > Brand & UX > Primary brand color hex.  We have color contrast systems under the hood that determine appropriate contrast for accessibility and readability.
